### PR TITLE
allow users to change fileTreeViwe/dirTreeView icon sizes.

### PR DESCRIPTION
**Pull request by krab**
Icon size change option via menu with saving/loading settings support, 24 by default.

### CODE REVIEW

**Comment by tsujan** [comment]
Thanks for your contribution!

Please use `QActionGroup` for this.

File: src/mainwindow.ui [hunk 1]
```diff
@@ -451,17 +462,37 @@
...
+   <property name="text">
+    <string>48 px</string>
+   </property>
+  </action>
  </widget>
  <customwidgets>
   <customwidget>
    <class>FileTreeView</class>
-   <extends>QTreeview</extends>
+   <extends>QWidget</extends>
```
Comment (tsujan) [inline src/mainwindow.ui:489]: Please restore QTreeview.

File: src/mainwindow.ui [hunk 2]
```diff
@@ -451,17 +462,37 @@
...
    <header>filetreeView.h</header>
    <container>1</container>
   </customwidget>
   <customwidget>
    <class>DirTreeView</class>
-   <extends>QTreeview</extends>
+   <extends>QWidget</extends>
```
Comment (tsujan) [inline src/mainwindow.ui:495]: Also, here.

File: src/mainwindow.ui [hunk 3]
```diff
@@ -116,19 +116,28 @@
...
     <addaction name="separator"/>
+    <addaction name="menuIcon_Size"/>
     <addaction name="actionFlatListMode"/>
     <addaction name="actionDirTreeMode"/>
     <addaction name="separator"/>
     <addaction name="actionExpand"/>
     <addaction name="actionCollapse"/>
     <addaction name="separator"/>
     <addaction name="actionStop"/>
-    <addaction name="actionReload"/>
```
Comment (tsujan) [inline src/mainwindow.ui:131]: Why did you remove this?

Comment (krab) [reply src/mainwindow.ui:131]: i didn't, its qtcreator doing some weird stuff, will fix.

**Comment by tsujan** [comment]
It's much better now.

Also,

 1. Just one slot, connected to `QActionGroup::triggered`, is enough.
 2. We don't need to save and read `QSize`. An `int` for the width is enough.
 3. The code doesn't take into account the changes in the width of the first column on changing the icon size, resulting in elided texts or unused spaces. Please fix that!

**Review by tsujan — APPROVED** [review]
Perfect! A nice feature to add.
